pydicom/charset.py

```diff
@@ -215,10 +227,11 @@ def _encode_to_jis_x_0208(value, errors='strict'):
             e.start = i
             e.end = len(value)
             raise e
-        if b[:3] == ENCODINGS_TO_CODES['iso8859']:
+        seq_length = 4 if b.startswith((b'\x1b$(', b'\x1b$)')) else 3
```

Comment:
seq_length doesn't appear to be used ... ?

Reply:
Thanks for your reviewing. You are exactly correct. I removed this line.